### PR TITLE
Skipping emptyDrop task in sn_rna mode

### PR DESCRIPTION
**Pull request by khajoue2**

### CODE REVIEW

File: dockers/skylab/loom-output/create_snrna_optimus.py [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+            gene_metric_values.append([np.nan] * ncols)
+
+    nrows = len(gene_ids)
+
+    # Generate row attribute dictionary
+    row_attrs = { "gene_names":gene_names, "ensembl_ids": gene_ids, "Gene": gene_names}
+
+    gene_metrics_data =np.array(gene_metric_values)
+    numeric_field_names = gene_metrics[0][1:]
+    for i in range(0, len(numeric_field_names)):
```
Comment (wjdingman1) [inline dockers/skylab/loom-output/create_snrna_optimus.py:112]:
```suggestion
    for i in range(len(numeric_field_names)):
```

File: dockers/skylab/loom-output/create_snrna_optimus.py [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+    final_df_non_boolean = final_df_non_boolean.apply(pd.to_numeric)
+
+    # COLUMN/CELL Metadata
+    col_attrs = dict()
+    col_attrs["cell_names"] = cell_ids
+    col_attrs["CellID"] = cell_ids
+    bool_field_names = final_df_bool_column_names
+
+    # Create metadata tables and their headers for bool
+    for i in range(0, bool_field_names.shape[0]):
```
Comment (wjdingman1) [inline dockers/skylab/loom-output/create_snrna_optimus.py:202]:
```suggestion
    for i in range(bool_field_names.shape[0]):
```

File: dockers/skylab/loom-output/create_snrna_optimus.py [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+    if args.verbose:
+        logging.info(
+            "shape of count matrix {0}, {1}".format(exp_counts["shape"], csr_exp_counts.shape)
+        )
+    nrows, ncols = csr_exp_counts.shape
+    expr_sp = sc.sparse.coo_matrix((nrows, ncols), np.float32)
+
+    xcoord = []
+    ycoord = []
+    value = []
```
Comment (wjdingman1) [inline dockers/skylab/loom-output/create_snrna_optimus.py:243]:
```suggestion
    xcoord = ycoord = value = []
   
```

**Review by wjdingman1 — APPROVED** [review]
looks good besides very minor syntax changes!

**Review by nikellepetrillo — APPROVED** [review]
i think this looks good!

**Pull request merged**